internal/pkg/cli/local_run.go

```diff
@@ -165,6 +178,22 @@ func (o *localRunOpts) Validate() error {
 		return fmt.Errorf("get application %s: %w", o.appName, err)
 	}
 	o.targetApp = app
+
+	// validate portOverrides
```

Comment:
Maybe make `portOverrides` or `portOverride` a type? And we don't need to use `split[0]`/`split[1]`